<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>        Aces</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>63</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>        Twos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>125</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>        Threes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>187</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>        Fours</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>496</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>     Full House</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>373</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>      Choice</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>249</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>        Fives</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>311</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>       Sixes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>682</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>       Yacht</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>434</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>     4 of a kind</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>744</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>       Total</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>558</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>   Little Straight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>620</y>
     <width>109</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>    Big Straight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acesButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>15</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="twosButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="threesButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="foursButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>205</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fivesButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>265</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sixesButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>327</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choiceButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fourofakindButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fullhouseButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>515</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="littlestraightButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>575</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bigstraightButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>635</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yachtButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>698</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="totalTextBrowser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>760</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="attackButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>760</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ATTACK!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
